tests/kw_env_test.sh

```diff
@@ -195,6 +198,40 @@ function test_use_target_env_invalid_env()
   assertEquals "($LINENO) Env does not exists" "$?" 22
 }
 
+function test_validate_env_before_use_invalid_case_with_config()
+{
+  local output
+
+  output=$(validate_env_before_use)
+  assertEquals "(${LINENO}) it should not allow switch to a new env due to .config file." "$?" 22
```

Comment:
I think `assert_equals_helper` is a better function for this type of assertion, because of the indentation and color of what is the expected value and what was the actual value.